File: BlogAuthService/AuthService.Core/Infrastructure/PageFilter/StreamPageFilter.cs
namespace AuthService.Core.Infrastructure.PageFilter
{
    /// <summary>
    /// Stream page filter.
    /// </summary>
    public class StreamPageFilter
    {
        /// <summary>
        /// Gets or sets the previous value.
        /// </summary>
        /// <value>
        /// The previous value.
        /// </value>
        public int PreviousValue { get; set; }

        /// <summary>
        /// Gets or sets the length.
        /// </summary>
        /// <value>
        /// The length.
        /// </value>
        public int Length { get; set; }

        /// <summary>
        /// Gets or sets the page count.
        /// </summary>
        /// <value>
        /// The page count.
        /// </value>
        public int PageCount { get; set; }
    }
}

File: Shared/Blog.FSharp.Core/Infrastructure/PageFilter/StreamPageFilter.fs
namespace Blog.FSharp.Core.Infrastructure.PageFilter

/// <summary>
/// Stream page filter.
/// </summary>
type StreamPageFilter() =
    /// <summary>
    /// Gets or sets previous value.
    /// </summary>
    member val PreviousValue: int = 0 with get, set

    /// <summary>
    /// Gets or sets length.
    /// </summary>
    member val Length: int = 0 with get, set

    /// <summary>
    /// Gets or sets page count.
    /// </summary>
    member val PageCount: int = 0 with get, set
